resources/CMakeLists.txt

```diff
@@ -245,6 +245,7 @@ set(energyplus_resources_src
   energyplus/ZoneBoundaryCondition/Bug_486_zone_bdr_test.idf
   energyplus/SmallOffice/SmallOffice.idf
   energyplus/AllFuelTypes/in.idf
+  energyplus/FrameAndDivider/in.idf
```

Comment:
Is this from an E+ example file? If so I would keep the name to make it clear which one that is

Reply:
Oh awesome :+1: thanks!